compoments/report-repair/repair-description.js

```diff
@@ -116,8 +117,14 @@ const RepairDescription = ({handleChange, values}) => {
       textError = 'Enter a description of the problem';
     }
     if (!textError && !imageError) {
+      var releaseVersion = '';
+      if (isMvpReleaseVersion()) {
+        releaseVersion = 'version-mvp'
+      } else if (isFullReleaseVersion()) {
+        releaseVersion = 'version-full'
+      }
       return handleChange('description', {
-        value: 'version-' + process.env.releaseVersion,        
+        value: releaseVersion,
```

Comment:
Please help me understand the need to provide the `releaseVersion` as part of the `value` to `handleChange`?

[handleChange](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/blob/dc26d1ea34c415bd6965dcc2e712188d7aeb8c6d/flow.js#L243) will store the `value` property in the state (and also determine the next page) and so I don't believe it's necessary to provide the `releaseVersion` as the `value`.

Reply:
@Bhavesh-Patel it's so we can define conditions in flow.js:

>  'repair-description': {prevStep: true, nextStep:[
>         {condition: 'version-full', nextStep: 'repair-image-upload'},
>         {condition: 'version-mvp', nextStep: 'contact-person'}
>       ]},

Or is there an alternative way around this?

Reply:
Given the use of global helper functions (i.e. nothing based on user interaction in this class), how about using the global helper function in flow.js (instead of here) to set to appropriate value for the `repair-description` `nextstep` value, i.e.
```
'contact-person': {prevStep: 'repair-description', nextStep: isMvpReleaseVersion()? 'contact-details' : 'repair-image-upload'},
```

Reply:
Have implemented this.